### ISSUE TITLE
sharks are cool!

### ISSUE THREAD

**Issue by BethMatta-NOAA** [nmfs-openscapes/2022-noaa-afsc-fall#8]
@jknielsen 

Would love to know more about the tagging work!

**Comment by jknielsen** [comment]
Yes I would love to tell you all about my recent sleeper shark study in Prince William Sound

**Comment by BethMatta-NOAA** [comment]
So awesome! Would love to hear about it!

**Comment by jknielsen** [comment]
I am going to make a list of things to talk about:

1. Tagging
2. Data

**Comment by EmilyMarkowitz-NOAA** [comment]
Sharks in PWS!? I want to hear about that! @bianca-prohaska-NOAA surely has cool stories, too!

![1048933119funny-shark-eats-pizze-animated-gif](https://user-images.githubusercontent.com/75965120/196806231-984349ae-1aef-45a4-bb1b-da6dcdb22746.gif)

**Comment by bianca-prohaska-NOAA** [comment]
Would be happy to share some eastern US shark stories!
